src/eth/block.yaml

```diff
@@ -70,3 +70,14 @@
     name: Uncle count
     schema:
       $ref: '#/components/schemas/uint'
+- name: eth_getBlockReceipts
+  summary: Returns the receipts of a block by number or hash.
+  params:
+    - name: Block
+      required: true
+      schema:
+        $ref: '#/components/schemas/BlockNumberOrTagOrHash'
+  result:
+    name: Receipts information
+    schema:
+      $ref: '#/components/schemas/ReceiptsInfo'
```

Comment:
Usually prefer to describe the scheme directly in the param/result if can be described simple and is not reused. So I think it is best to update this to directly be the array of `ReceiptInfo`.

Reply:
thanks for the quick comment, I'll change it right now.